docs/single-to-multi-master.md

```diff
@@ -91,25 +104,55 @@ reference a third one, even if we have not created it yet.*
 $ kops edit cluster example.com
 ```
 
- * In ``.spec.etcdClusters`` 2 new members in each cluster, one for each new
+ * In ``.spec.etcdClusters`` add 2 new members in each cluster, one for each new
  availability zone.
 
 ```yaml
     - instanceGroup: master-<availability-zone2>
-      name: <availability-zone2>
+      name: <availability-zone2-name>
     - instanceGroup: master-<availability-zone3>
-      name: <availability-zone3>
+      name: <availability-zone3-name>
 ```
 
+Example:
+
+```yaml
+etcdClusters:
+  - etcdMembers:
+    - instanceGroup: master-eu-west-1a
+      name: a
+    - instanceGroup: master-eu-west-1b
+      name: b
+    - instanceGroup: master-eu-west-1c
+      name: c
+    name: main
+  - etcdMembers:
+    - instanceGroup: master-eu-west-1a
+      name: a
+    - instanceGroup: master-eu-west-1b
+      name: b
+    - instanceGroup: master-eu-west-1c
+      name: c
+    name: events
+```
+
+
 ## 3 - Add a new master
 
 ### a - Add a new member to the etcd clusters
 
 **The clusters will stop to work until the new member is started**.
 
 ```bash
-$ kubectl --namespace=kube-system exec etcd-server-ip-172-20-36-161.ec2.internal -- etcdctl member add etcd-<availability-zone2> http://etcd-<availability-zone2>.internal.example.com:2380
-$ kubectl --namespace=kube-system exec etcd-server-events-ip-172-20-36-161.ec2.internal -- etcdctl --endpoint http://127.0.0.1:4002 member add etcd-events-<availability-zone2> http://etcd-events-<availability-zone2>.internal.example.com:2381
+$ kubectl --namespace=kube-system exec etcd-server-ip-172-20-36-161.ec2.internal -- etcdctl member add etcd-<availability-zone2-name> http://etcd-<availability-zone2-name>.internal.example.com:2380 \
```

Comment:
Nit picks that you can follow up in another or if you like. Looks grear btw

I hate, and I know I have done it before the ‘$’ sign with the prompt makes copy paste a pain.

Can we get the name of the pod via a label? That way we do not have to have a name the is unique in the examples.